index.js

```diff
@@ -0,0 +1,99 @@
+const AWS = require('aws-sdk');
+const { config } = require('@firstandthird/arc-rapptor');
```

Comment:
I think we should make it a class and you pass in bucket and delimiter as options, that way it doesn't depend on arc-rapptor